### PR TITLE
New package: Wondershare.Anireel version 1.1.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/32824)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by TaylorShi — COMMENTED** [review]
OK

File: manifests/w/Wondershare/Anireel/1.1.2/Wondershare.Anireel.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.wondershare.com/cbs_down/anireel_full9589.exe
```
Comment (ItzLevvie) [inline manifests/w/Wondershare/Anireel/1.1.2/Wondershare.Anireel.installer.yaml:16]:
```suggestion
  InstallerUrl: https://download.wondershare.com/cbs_down/anireel_64bit_full9589.exe
```
If you want it to install `1.1.2` then probably use https://download.wondershare.com/cbs_down/anireel_64bit_full9589.exe?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/w/Wondershare/Anireel/1.1.2/Wondershare.Anireel.installer.yaml (hunk 1 above)
Comment (vedantmgoyal2009) [reply manifests/w/Wondershare/Anireel/1.1.2/Wondershare.Anireel.installer.yaml]: From where did you get this?

**Review by ItzLevvie — APPROVED** [review]
LGTM:
image

**Review by OfficialEsco — APPROVED** [review]

File: manifests/w/Wondershare/Anireel/1.1.2/Wondershare.Anireel.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/w/Wondershare/Anireel/1.1.2/Wondershare.Anireel.installer.yaml]: It's the offline installer name from your screenshot: anireel_64bit_full9589.exe

**Pull request merged**